packages/babel-preset-default/index.js

```diff
@@ -1,36 +1,75 @@
 module.exports = function( api ) {
+	let wpBuildOpts = {};
 	const isTestEnv = api.env() === 'test';
+	const isWPBuild = ( name ) => [ 'WP_BUILD_MAIN', 'WP_BUILD_MODULE' ].some(
+		( buildName ) => name === buildName
+	);
+
+	// Because we serve a different preset depending
+	// on the caller options, we need to tell the cache
+	// when it needs updating. Otherwise, it won't be
+	// recalculated for different builds.
+	api.cache.using( () => isWPBuild( wpBuildOpts.name ) );
+
+	api.caller( ( caller ) => {
+		if ( caller && isWPBuild( caller.name ) ) {
+			wpBuildOpts = { ...caller };
+		}
+	} );
+
+	const getPresetEnv = () => {
+		const opts = {};
+
+		if ( isTestEnv ) {
+			opts.useBuiltIns = 'usage';
+		} else {
+			opts.modules = false;
```

Comment:
```suggestion
			opts.modules = isWPBuild( wpBuildOpts.name ) ? wpBuildOpts.module : false;
```

Reply:
That would change the current logic. We want `modules=commonjs` independently of what `isTestEnv` value is.